### PR TITLE
Fix analyzer error in appender_test.dart

### PR DESCRIPTION
**Pull request by davidmorgan**

### CODE REVIEW

File: test/appender_test.dart [hunk 1]
```diff
@@ -37,7 +37,7 @@ class SimpleLogger implements Logger {
   StreamController<LogRecord> _controller = new StreamController(sync:true);
   Stream<LogRecord> get onRecord => _controller.stream;
 
-  void info(String msg, [Object message, StackTrace stackTrace]) =>
+  void info(msg, [Object message, StackTrace stackTrace]) =>
```
Comment (jtmcdole) [inline test/appender_test.dart]: "message" is a String in LogRecord:
https://github.com/dart-lang/logging/blob/46077ac18c06c2ff58a8053c55a61ef3802e7a07/lib/logging.dart#L362

Comment (davidmorgan) [reply test/appender_test.dart]: But the type here is Logger, not LogRecord :)

https://github.com/dart-lang/logging/blob/46077ac18c06c2ff58a8053c55a61ef3802e7a07/lib/logging.dart#L212

Comment (jtmcdole) [reply test/appender_test.dart]: My confusion stems from this not being named "error". Maybe we could change that as well as the type?

Comment (davidmorgan) [reply test/appender_test.dart]: I guess you mean to make the arg names match the method it's overriding? Sure thing, done.

**Comment by davidmorgan** [comment]
Friendly ping, PTAL.

**Pull request merged**